run-tests

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+EXPECT_5LVL=0
+SUPPORT_NO5LVL=0
+SUPPORT_LA57=0
+SUPPORT_5LVL=0
```

Comment:
Suggestion: rename `SUPPORT_5LVL` to `SUPPORT_KERNEL_5LVL` or `SUPPORT_CONFIG_5LVL` to indicate that its a kernel configuration option.